doc/lsst.daf.butler/queries.rst

```diff
@@ -441,23 +442,26 @@ Few examples of valid expressions using some of the constructs:
 Query result ordering
 ---------------------
 
-Few query methods (`~Registry.queryDataIds` and `~Registry.queryDimensionRecords`) support special constructs for ordering and limiting the number of the returned records. These methods return iterable objects which have ``order_by()`` and ``limit()`` methods. Methods modify the iterable object and should be used before iterating over resulting records, for convenience the methods can be chained, see example below.
+Butler query methods (`Butler.query_datasets`, `Butler.query_data_ids`, and
+`Butler.query_dimension_records`) support ordering and limiting the number of
+the returned records. These methods have ``order_by`` and ``limit``
+parameters controlling this behavior.
 
-The ``order_by()`` method accepts a variable number of positional arguments specifying columns/fields used for ordering, each argument can have one of the supported formats:
+The ``order_by`` parameter accepts a list of strings specifying columns/fields used for ordering. Each string can have one of the supported formats:
 
 - A dimension name, corresponding to the value of the dimension primary key, e.g. ``"visit"``
 - A dimension name and a field name separated bey a dot. Field name can refer to any of the dimension's metadata or key, e.g. ``"visit.name"``, ``"detector.raft"``. Special field names ``"timespan.begin"`` and ``"timespan.end"`` can be used for temporal dimensions (visit and exposure).
 - A field name without dimension name, in that case field is searched in all dimensions used by the query, and it has to be unique. E.g. ``"cell_x"`` means the same as ``"patch.cell_x"``.
 - To reverse ordering for the field it is prefixed with a minus sign, e.g. ``"-visit.timespan.begin"``.
 
-The ``limit()`` method accepts two positional integer arguments - limit for the number of returned records and offset (number of records to skip). The offset argument is optional, if not provided it is equivalent to offset 0.
+The ``limit`` parameter accepts an integer specifying the maximum number of results to return.
```

Comment:
Maybe mention the difference between positive and negative limit and what happens by default if no results are returned (but limit=0 does not complain for no results since you asked for none).

Reply:
I considered that originally but decided it was Too Much Information for an overview document -- the parameter description into the API documentation covers that in detail already.